server/utils/zip.js

```diff
@@ -3,13 +3,28 @@ import fs from 'fs';
 import JSZip from 'jszip';
 import tmp from 'tmp';
 import unescape from '../../shared/utils/unescape';
-import { Collection, Document } from '../models';
+import { Attachment, Collection, Document } from '../models';
+import { getImageByKey } from './s3';
 
 async function addToArchive(zip, documents) {
   for (const doc of documents) {
     const document = await Document.findByPk(doc.id);
+    let text = unescape(document.text);
 
-    zip.file(`${document.title}.md`, unescape(document.text));
+    const attachments = await Attachment.findAll({
```

Comment:
This was quite nice as it let me get rid of the regex and use exact replace methods. Should be better going forward.

Reply:
Nice! This does mean that only image attachments created from this point on would be returned inside the archive, any existing attachments would not – but that's fine by me.

Reply:
probably also fine because anything before this change would not have an entry in the new attachments table anyway.